internal/subscriber/ordering.go

```diff
@@ -132,6 +132,15 @@ func (s *OrderedImplementation) Pull(ctx context.Context, req *PullRequest, resp
 		ts.Seconds = msg.PublishTime.Unix()
 		protoMsg.PublishTime = ts
 
+		if len(protoMsg.Attributes) == 0 {
+			protoMsg.Attributes = make(map[string]string, len(msg.Attributes))
```

Comment:
If we're only assigning `UberTraceID` to the map, can we avoid initializing one with the size of `msg.Attributes`?